Remove.py

```diff
@@ -1,35 +1,48 @@
 import PrintList
 
 
-def remove_1(to_Do, removed):
+def remove_1(to_Do):
 
   print("""
   ------------------------------
   Remove an item
   ------------------------------
   """)
-  
+
+  # Print current list
   PrintList.print_list(to_Do)
 
-  X = input("[Enter X to exit this screen]")
+  #User presses X to exit screen
+  X = input("[Enter X to exit this screen]").upper()
 
-  item_to_remove = int(input("What item would you like to remove?")) - 1
+  if X == "X":
+    return
+  else:
+    pass
```

Comment:
We can remove this `else` statement.

Reply:
To clarify why I'm saying we can remove this - since the else condition here is just "let the program continue" there's no need to include that. If the `if` check does not pass (the user has not entered X) the program would continue anyway, thus this `else` turns out to be redundant. Less work for us to have to type out elses when they're not needed!